harness/determined/pytorch/_experimental.py

```diff
@@ -28,6 +29,29 @@ def use_amp(self) -> None:
         self._parent.wrap_scaler(amp.GradScaler())  # type: ignore
         self._auto_amp = True
 
+    def disable_auto_to_device(self) -> None:
```

Comment:
do you mean only called from `__init__` for a trial?  i'm not sure that we should enforce it since we don't enforce for other modifiers like `use_amp`